components/script/dom/serviceworkerglobalscope.rs

```diff
@@ -419,7 +418,11 @@ impl ServiceWorkerGlobalScope {
                         ports,
                     );
                 } else {
-                    MessageEvent::dispatch_error(target, scope.upcast());
+                    ExtendableMessageEvent::dispatch_error(
+                        target,
+                        scope.upcast(),
+                        message.handle(),
```

Comment:
When we will be using `ExtendableMessageEvent::MessageEventInit::empty()`, we don't need to pass the `HandleValue` anymore, since it will be part of the empty initialized value.

Reply:
Okay, for some reason I thought we needed the data for the error handling. I apologize I'm newer to the code base. I'll get this fixed right away. Thank you for the input!